src/components/editor/InputLiteral.jsx

```diff
@@ -22,9 +22,33 @@ export class InputLiteral extends Component {
     this.notRepeatable = this.notRepeatable.bind(this)
     this.addUserInput = this.addUserInput.bind(this)
     this.state = {
-      content_add: ""
+      content_add: "",
+      defaults: []
     }
     this.lastId = -1
+
+    let defaultValue
```

Comment:
Do we need the `let` statement here? It looks as though `defaultValue` is only used in the `try` block.

Reply:
You're right, I guess as a bit of defensive programming, I thought it would barf on setting that variable if there were no `valueConstraint.defaults[0]` but it appers that is not the case. I also did this same thing in the InputListLOC component, so I changed that as well. Also, since the variable is not mutating it can be a `const` I think.

Reply:
👍